changelog/next/changes/5113--docker-trixie.md

```diff
@@ -0,0 +1,2 @@
+We have upgraded our Docker baseimage from `debian:bookwork-slim` to
+`debian:trixie-slim`.
```

Comment:
The `tenzir/tenzir` and `tenzir/tenzir-node` Docker images now use `debian:trixie-slim` instead of `debian:bookworm-slim` as a base image.